### PR TITLE
Update cm-emmc-flashing.md

### PR DESCRIPTION
**Pull request by timg236**
@dp111 - I think this covers the "how to enable network boot" FAQ. For complicated boot order / failovers I think people will need to take dive into the full configuration guide

### CODE REVIEW

File: hardware/computemodule/cm-emmc-flashing.md [hunk 1]
```diff
@@ -118,25 +118,36 @@ The `/dev/sdX1` and `/dev/sdX2` partitions can now be mounted normally.
...
 
-```bash
-./rpiboot -d recovery
-```
+* Selecting and verifying a specific bootloader release. The version in the `usbboot` repo is always a recent stable release.
+* Configuring the boot device (e.g. network boot). See `BOOT_ORDER` section in the [bootloader configuration](raspberrypi/bcm2711_bootloader_config.md) guide.
+* Enabling hardware write protection on the bootloader EEPROM to ensure that the bootloader can't be modified on remote/inaccessible products.
 
-The `recovery` directory of the `rpiboot` tool contains a default `pieeprom.bin` bootloader EEPROM image. See the [boot EEPROM](../raspberrypi/booteeprom.md) and [bootloader configuration](../raspberrypi/bcm2711_bootloader_config.md) pages for more information about how to change the embedded configuration file.
+N.B. The Compute Module 4 ROM never runs `recovery.bin` from SD/EMMC and the `rpi-eeprom-update` services is not enabled. This is necessary because the EMMC is not removable and an invalid `recovery.bin` file would prevent the system from booting.
```
Comment (peterharperuk) [inline hardware/computemodule/cm-emmc-flashing.md:129]: "services is not enabled" should be "services are not enabled"?

Comment (timg236) [reply hardware/computemodule/cm-emmc-flashing.md:129]: Should be "service" singular

File: hardware/computemodule/cm-emmc-flashing.md [hunk 2]
```diff
@@ -118,25 +118,36 @@ The `/dev/sdX1` and `/dev/sdX2` partitions can now be mounted normally.
...
-```bash
-sha256sum pieeprom.bin | awk '{print $1}' > pieeprom.sig
-````
+* Replace `recovery/pieeprom.original.bin` if a specific bootloader release is required.
+* Edit the default `recovery/boot.conf` bootloader configuration file. Typically, at least the BOOT_ORDER must be updated:-
+   * For network boot `BOOT_ORDER=0xf2`
+   * For SD/EMMC boot `BOOT_ORDER=0xf1`
+   * For USB boot failing over to EMMC `BOOT_ORDER=0xf15`
+* Run `recovery/update-pieeprom.sh` to update the EEPROM image `pieeprom.bin` image file. 
+* If EEPROM write protection is requried then edit `recovery/config.txt` and add `eeprom_write_protect=1`. Hardware write-protection is must be enabled via software and then locked by pulling the `EEPROM_nWP` pin low.
```
Comment (peterharperuk) [inline hardware/computemodule/cm-emmc-flashing.md:140]: Remove "is" in this sentence: Hardware write-protection is must

**Comment by timg236** [comment]
@ghollingworth

**Comment by timg236** [comment]
@lurch @dp111 Any chance it can be merged - I think some people would benefit from the simplified instructions

**Comment by dp111** [comment]
looks good to me , a couple of typos , "  overriden", "  requried"

**Pull request merged**